.github/ISSUE_TEMPLATE.md

```diff
@@ -1,80 +1,38 @@
-## 1. General summary of the issue
-
 <!--
 
-Please provide a descriptive title of the issue in the field 'Title' too.
+Please provide a descriptive title in the field 'Title' too.
 
-Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore.
+Please be aware that Pester version 3.4.0 - which is shipped with Windows 10 and Windows Server 2016 - is not supported anymore. Please update Pester to the latest version. See [Installation and update guide](https://pester.dev/docs/introduction/installation).
 
-Please update Pester - before submitting a bug report - and retest your code under the newest version of Pester.
+-->
 
-[Full installation and update guide](https://github.com/pester/Pester/wiki/Installation-and-Update).
+## Question
 
--->
+<!-- 
 
-## 2. Describe Your Environment
+Please be clear and concise as it will help us answer you faster.
 
-<!--
+Provide sample code and output if it helps - use code blocks like below.
 
-If you would like to submit a bug report, please provide the output of a code provided below.
+```powershell
+    #My code or output
+```
 
-If you would like to write about anything else - like a feature request - feel free to remove a provided template text.
+-->
+
+
+**Environment data**
+
+<!--Please provide the output of a code provided below.
 
 Operating System, Pester version, and PowerShell version:
 
-```powershell
 $bugReport = &{
-    $p = Get-Module -Name Pester -ListAvailable | Select-Object -First 1
+    $p = Get-Command Invoke-Pester | Select-Object -ExpandProperty Module
```

Comment:
This will work only if Pester is imported. The original solution will show what would be automatically imported. This is trade off between succeeding in when this is run in a clean session, and being more accurate. I would use the current solution. Or we can move the script to a gist and consider both options. 

Provide the result of this script:
```powershell
(Invoke-WebRequest -Uri "https://git.io/JTinj" -UseBasicParsing).Content | Invoke-Expression
```
This will run this gist: https://gist.github.com/nohwnd/3c78fb0e452dd4a49a8810c155023e33

Or run it manually:

```powershell
$bugReport = & {
    $i = Get-Command Invoke-Pester -ErrorAction SilentlyContinue | Select-Object -ExpandProperty Module
    $m = if ($i) { $i } else { Get-Module -Name Pester -ListAvailable | Select-Object -First 1 }
    $psv = $PSVersionTable.PSVersion
    $pre = $m.PrivateData -and $m.PrivateData.PSData -and $m.PrivateData.PSData.PreRelease
    $pv = if ($pre) { "$($m.Version)-$($m.PrivateData.PSData.PreRelease)" } else { $m.Version }
    $pi = if ($i) { " (available)" } else { " (imported)"} 
    "Pester version     : " + $pv + " " + $m.Path + $pi
    "PowerShell version : " + $psv
    "OS version         : " + [System.Environment]::OSVersion.VersionString
}
$bugReport
$bugReport | clip
```

Reply:
`Get-Command Invoke-Pester` will autoload the same version you get from the failsafe if it's not already loaded. At the same time it works with manually imported module ex. from bin folder in git-repo. Or am I missing something?

I really like that gist-solution though! Do we think users (who don't open the link to read) would find it scary? I know similar is used for installation-guides in other Powershell modules, vscode ++, so might be good.

Reply:
Btw. last line needs to be removed. "clip" is windows-only and "Set-Clipboard" is PSv5+

Reply:
Updated the gist. Removed clip and reverted to your approach. The shortlink seems to be pointing to a cached version that is in the middle of the changes, hopefully this will go away automatically.